scripts/test_integration_pairs.py

```diff
@@ -0,0 +1,165 @@
+import os
+import tempfile
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from itertools import combinations
+from subprocess import PIPE, run
+from typing import Any, Dict, Tuple
+
+from rich import print
+
+from zenml.integrations.registry import integration_registry
+
+
+def get_integration_requirements(integration_name: str) -> str:
+    """Get the requirements for a given integration.
+
+    Args:
+        integration_name: The name of the integration.
+
+    Returns:
+        The requirements for the integration as a string.
+    """
+    requirements = integration_registry.select_integration_requirements(
+        integration_name
+    )
+    if integration_name == "prodigy":
+        requirements = [req for req in requirements if req != "prodigy"]
```

Comment:
Why `prodigy` is so specific here?

Reply:
There's no way to actually install prodigy using `zenml integration install` because you need to download the wheel directly from explosion with a user key... So we ignore it here.